docs/clients/http/protocol.rst

```diff
@@ -36,8 +38,9 @@ submit the following JSON-encoded form with the necessary fields::
 Response
 --------
 
-The response format is the same for both methods. The body of the
-response is JSON of the following form::
+The format of a request's response is the same for both methods. The
+``Content-Type`` of a response is returned in ``application/json`` with the
+following form::
```

Comment:
I don't like the new wording, IMO the original one is better. 

> The ``Content-Type`` of a response is returned in ``application/json``

This reads as an incorrect statement. Content-type is an HTTP header, it's returned in ASCII, and can indicate a JSON payload when it's set to "application/json". None of which is relevant to the current documentation page, we're not documenting HTTP.